client/tso_client.go

```diff
@@ -178,6 +178,14 @@ func (c *tsoClient) getTSORequest(ctx context.Context, dcLocation string) *tsoRe
 	return req
 }
 
+func (c *tsoClient) getTSODispatcher(dcLocation string) (*tsoDispatcher, bool) {
```

Comment:
does this function need to be put in `tso_dispatcher.go`?

Reply:
Here is the explain: https://github.com/tikv/pd/pull/8121#discussion_r1575885704